### PR TITLE
Address noisy alerts (https-macos, hidden exec, bytes written, dev entries, long running)

### PR DESCRIPTION
**Pull request by tstromberg**
- unexpected-https-macos: add extra tag
- high-disk-bytes-written: add extra tag
- hidden executable: add more generic patterns to decrease whack-a-mole
- unexpected dev entries: fix edge bug
- long running framework: add nix exception
- recently created, long running: refactor

### CODE REVIEW

File: detection/evasion/hidden-executable.sql [hunk 1]
```diff
@@ -125,6 +119,8 @@ WHERE (
     f.path LIKE '/nix/store/%'
     AND p0.name LIKE '%-wrappe%'
   )
-  AND NOT f.path LIKE '%/.Trash/1Password %.app/Contents/Library/LoginItems/1Password Extension Helper.app/Contents/MacOS'
-  AND NOT f.path LIKE '/home/%/.local/share/AppImage/ZenBrowser.AppImage'
+  AND NOT homedir LIKE '~/.Trash/1Password %.app/Contents/Library/LoginItems/1Password Extension Helper.app/Contents/MacOS'
+  AND NOT homedir LIKE '~/.local/share/AppImage/ZenBrowser.AppImage'
+  AND NOT homedir LIKE '~/Library/Application Support/Code/User/globalStorage/ms-dotnettools.vscode-dotnet-runtime/.dotnet/%'
+  AND NOT homedir LIKE '%/.Trash/1Password %.app/Contents/Library/LoginItems/1Password Extension Helper.app/Contents/MacOS'
```
Comment (egibs) [inline detection/evasion/hidden-executable.sql:125]: This is a dupe of L122.

**Review by egibs — COMMENTED** [review]
In `unexpected-talkers-macos.sql` I was going to update the `gvproxy` query at the bottom to:
```sql
AND NOT (
    unsigned_exception IN (
      '500,6,32768,gvproxy,gvproxy',
      '500,17,123,gvproxy,gvproxy'
    )
    AND p0.path LIKE '/opt/homebrew/Cellar/podman/%/libexec/podman/gvproxy'
  )
```
https://github.com/chainguard-dev/osquery-defense-kit/blob/12077261e734a3968b6df5eb10fd1f6889dc7b2a/detection/c2/unexpected-talkers-macos.sql#L105-L108

**Comment by egibs** [comment]
Here are the other changes I was going to add for reference (some of which are covered here): https://github.com/chainguard-dev/osquery-defense-kit/compare/main...egibs:20241030-exceptions?expand=1

**Review by r0cketlad — APPROVED** [review]

**Comment by tstromberg** [comment]
@egibs - those changes sound good and shouldn't conflict. I've merged this PR and will  be happy to review your additions.

**Pull request merged**